src/SmartComponents/RunTaskModal/InputParameters.js

```diff
@@ -46,20 +30,23 @@ export const InputParameter = ({ parameter, setDefinedParameters }) => {
   return (
     <Form className="pf-v5-u-pb-lg">
       <FormGroup
-        label={parameter.key}
+        label={parameter.title || parameter.key}
```

Comment:
Why have we added `title` here? I don't see title as a field in the api return. I want to make sure I'm not missing something.

Reply:
I pre-emptively put it in for when the accompanying API MR is merged: https://gitlab.cee.redhat.com/insights-platform/advisor-backend/-/merge_requests/872

I was just being lazy putting the dropdown and title in the same commit.  I can put it in a separate commit if you want to separate the different issues.